src/sas/qtgui/Utilities/UI/DocViewWidgetUI.ui

```diff
@@ -44,19 +44,6 @@
        </property>
       </spacer>
      </item>
```

Comment:
Rather than deleting this, you could just add
```python
self.editButton.setVisible(False)
```
 in `DocViewWidget.py`
It will be easier than restoring the deleted code later.